docs/designers-developers/developers/slotfills/plugin-preview-menu-item.md

```diff
@@ -0,0 +1,37 @@
+# PluginPreviewMenuItem
+
+This is designed to be used alongside the PluginPreview.
+
+-   PluginPreviewMenuItem: Adds a menu item to the "Preview" menu.
+-   PluginPreview: Renders the main content area when that menu item is chosen.
+
+Each of these takes 2 props, `deviceName`, and `children`.
+
+-   `deviceName` - A string that serves as an internal identifier for your
```

Comment:
I think I'd keep this more generic and not assume it's a device.

Think e.g. AMP previews, social media previews, etc

Reply:
I changed the external interface to use `previewId` as a way to match the menu item with the content.